2022-10-19-recommenders-1.md

```diff
@@ -15,9 +15,9 @@ If you want to receive reminders about this event, sign up here
 
 ## Technologies 
 
-* Tech 1
-* Tech 2
-* Tech 3
+* Scikit-Learn
```

Comment:
Do you think scikit learn will be enough for CF recommenders? What about tools like implicit?

Reply:
I agree, I'll add Implicit and Surprise.